CoffeeChat/Controller/ConnectSocialMediaViewController.swift

```diff
@@ -0,0 +1,151 @@
+//
+//  ConnectSocialMediaViewController.swift
+//  CoffeeChat
+//
+//  Created by Amy Chin Siu Huang on 10/18/20.
+//  Copyright © 2020 cuappdev. All rights reserved.
+//
+
+import UIKit
+
+class ConnectSocialMediaViewController: UIViewController {
+    
+    // MARK: - Private View Vars
+    private let backButton = UIButton()
+    private let backgroundImage = UIImageView(frame: UIScreen.main.bounds)
+    private let fbTextField = UITextField()
+    private let infoTextView = UITextView()
+    private let instaTextField = UITextField()
+    private let saveBarButtonItem = UIBarButtonItem()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .backgroundLightGreen
+        setupNavigationBar()
+        
+        backgroundImage.image = UIImage(named: "aboutPearBackground")
```

Comment:
could we rename this image asset if it's not used solely for the about pear vc's background?